app/shared/resource-list/ResourceListItem.js

```diff
@@ -26,7 +28,20 @@ class ResourceListItem extends Component {
             height={420}
             image={getMainImage(resource.images)}
             width={700}
-          />
+          >
+            <Label
+              className="app-ResourceListItem__peopleCapacity"
+              shape="rounded"
+              size="mini"
+              theme="orange"
+            >
+              {resource.peopleCapacity}
+              <FontAwesome name="user" />
```

Comment:
- [ ] We could maybe use the icon "users" for people capacity even though it differs from the designs. But would represent number of people better imo.